forwarding/src/test/kotlin/com/pkware/java/nio/file/forwarding/ForwardingFileSystemTest.kt

```diff
@@ -0,0 +1,32 @@
+package com.pkware.java.nio.file.forwarding
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
```

Comment:
Take a look at using `org.mockito.kotlin:mockito-kotlin:4.0.0`.

Reply:
ended up not having to use mockito as we can use jimfs